src/main/java/com/meetup/server/global/support/jwt/JwtTokenProvider.java

```diff
@@ -0,0 +1,79 @@
+package com.meetup.server.global.support.jwt;
+
+import com.meetup.server.auth.dto.CustomOAuth2User;
+import com.meetup.server.auth.dto.response.JwtUserDetails;
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.security.Keys;
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.security.Key;
+import java.util.Date;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class JwtTokenProvider {
+    private final JwtProperties jwtProperties;
+    private Key key;
+
+    @PostConstruct
+    public void setKey() {
+        this.key = Keys.hmacShaKeyFor(jwtProperties.secretKey().getBytes());
+    }
+
+    public String createAccessToken(Object member) {
+        return createToken(member, jwtProperties.accessTokenExpiration());
```

Comment:
```suggestion
        public String createAccessToken(Object user) {
            return createToken(user, jwtProperties.accessTokenExpiration());
```

아래 `createRefreshToken()`, `createToken()` 도 동일하게 수정해주세요!